scripts/Recovery/SampleRecoverySave.py

```diff
@@ -0,0 +1,17 @@
+from mantid.simpleapi import *
```

Comment:
Was this file meant to be added as it looks a bit like some example code or a test? For samples can we add an `examples` directory to split of functional code and example scripts?